src/node/txreconciliation.cpp

```diff
@@ -51,9 +59,6 @@ class TxReconciliationState
     bool m_we_initiate;
 
     /**
-     * TODO: These fields are public to ignore -Wunused-private-field. Make private once used in
-     * the following commits.
-     *
```

Comment:
nit: if you have created a refactor commit to deal with removing the TODOs, it may be worth moving that change there.

Not sure if you're planning to squash it or not. Feel free to disregard.